jetty-ee10/jetty-ee10-demos/jetty-ee10-demo-jaas-webapp/src/main/config/modules/ee10-demo-jaas.mod

```diff
@@ -20,7 +20,6 @@ ext
 
 [files]
 basehome:modules/demo.d/ee10-demo-jaas.xml|webapps/ee10-demo-jaas.xml
-basehome:modules/demo.d/ee10-demo-login.conf|etc/ee10-demo-login.conf
```

Comment:
Why this? 

All the config files necessary were in the `demo.d` directory and copied to relevant places.
Why is now `ee10-demo-login.conf` an exception?

Reply:
Also, the `ee10-demo-login-conf` file still references module `xyz` but should be `ee10-xyz`.

Reply:
I thought I made the problem clean in the opening comment of the PR? The problem is that you cannot have more than one jaas config file, so all of the eeX-demo-login.conf files were being ignored. All of the jaas configuration must go into a single file.